### PR TITLE
VASP main page updates

### PR DESCRIPTION
**Pull request by tayloraubry**
Ported VASP friendly users documentation to main page and cut down text overall

### CODE REVIEW

File: docs/Documentation/Applications/vasp.md [hunk 1]
```diff
@@ -347,17 +328,30 @@ To run VASP, the following 4 input files are needed: POSCAR, POTCAR, INCAR, KPOI
...
+
+    On Swift, VASP is most efficiently run on partially full nodes. 
+
+    Unlike on Eagle, multiple jobs can run on the same nodes on Swift. If you are only using a fraction of a node, other users' jobs could be assigned to the rest of the node, which might deteriorate the performance. Setting "#SBATCH --exclusive" in your run script prevents other users from using the same node as you, but you will be charged the full 5AUs/node, regardless of the number of CPUs/node you are using. 
+
+## VASP on Vermilion
 
-??? example "Vermilion: VASP 6 (Intel MPI) on CPUs"
+#### CPU
+??? example "Sample job script: Vermillion - VASP 6 (Intel MPI)"
```
Comment (mcfarljm) [inline docs/Documentation/Applications/vasp.md:354]: I was not able to run this script.  As written, I get an error loading the vasp module:
```bash
[jmcfarla@vs-login-1 vasp-docs]$ module purge
[jmcfarla@vs-login-1 vasp-docs]$ ml vasp/6.3.1
Lmod has detected the following error: The following module(s) are unknown: "vasp/6.3.1"
```

I tried to work around this by using:
```bash
source /nopt/nrel/apps/210929a/myenv.2110041605
ml vaspintel
```
And reducing the number of tasks from 60 to 59.

However, even with those changes I get a slew or slurm errors, starting with:
```
slurmstepd: error: mpi/pmi2: invalid client request
slurmstepd: error: mpi/pmi2: request not begin with 'cmd='
slurmstepd: error: mpi/pmi2: full request is: 000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```

I don't want to delay getting the rest of the updates from this PR merged, but I think at a minimum we should open a ticket about getting a working example script.

File: docs/Documentation/Applications/vasp.md [hunk 2]
```diff
@@ -452,7 +446,27 @@ To run VASP, the following 4 input files are needed: POSCAR, POTCAR, INCAR, KPOI
...
+    module load openmpi
+    OMPI_MCA_param="btl_tcp_if_include ens7"
+    ```
+
+#### GPU
+
+??? example "Sample job script: Vermillion - VASP 6 (OpenACC)"
   
     ```
     #!/bin/bash
```
Comment (mcfarljm) [inline docs/Documentation/Applications/vasp.md:472]: Has the module for this example changed?  I get an error loading the module (with or without sourcing the standard /nopt/nrel/apps env).

```
[jmcfarla@vs-login-1 vasp-docs]$ ml vasp/6.3.1-nvhpc_acc
Lmod has detected the following error: The following module(s) are unknown: "vasp/6.3.1-nvhpc_acc"
[jmcfarla@vs-login-1 vasp-docs]$ source /nopt/nrel/apps/210929a/myenv.2110041605                                  
[jmcfarla@vs-login-1 vasp-docs]$ ml vasp/6.3.1-nvhpc_acc
Lmod has detected the following error: The following module(s) are unknown: "vasp/6.3.1-nvhpc_acc"
```

File: docs/Documentation/Applications/vasp.md [hunk 3]
```diff
@@ -419,7 +413,7 @@ To run VASP, the following 4 input files are needed: POSCAR, POTCAR, INCAR, KPOI
 
     srun --mpi=pmi2 -n 60 vasp_std
     ```
-??? example "Vermilion: VASP 5 (Intel MPI) on CPUs"
+??? example "Sample job script: Vermillion - VASP 5 (Intel MPI)"
```
Comment (mcfarljm) [inline docs/Documentation/Applications/vasp.md:416]: I get an error when trying to load the module in this example:

```
[jmcfarla@vs-login-1 vasp-docs]$  module purge
[jmcfarla@vs-login-1 vasp-docs]$  ml vasp/5.4.4
Lmod has detected the following error: These module(s) or extension(s) exist but cannot be loaded as requested:
```

**Review by mcfarljm — COMMENTED** [review]
I think the reorganization and simplification is a great improvement.

My main comments are:

1. Would it be best to remove the VASP sections that currently exist under Swift (https://nrel.github.io/HPC/Documentation/Systems/Swift/running/#running-vasp) and Vermilion (https://nrel.github.io/HPC/Documentation/Systems/Vermilion/running/#running-vasp-on-vermilion), and perhaps replace those with a link to this section, since you have provided examples for all systems here?  At a minimum, that reduces duplication and would make it easier to maintain the example scripts moving forward.
2. I had problems running the example scripts for Vermilion, per my comments.  I know that's not the focus of this PR, so I wouldn't want to delay getting these changes incorporated just because of the issues with the vermilion examples.  If there aren't quick fixes for the vermilion examples, we should probably create a GH Issue to keep track of getting them updated.

**Comment by yandthj** [comment]
Thanks for the review @mcfarljm . I have a PR in (https://github.com/NREL/HPC/pull/501) to remove the VASP sections under Swift and Vermilion pages as you mentioned in your comment. 
@tayloraubry If there aren't quick fixes for the vermilion example problems that John pointed out, could you create issues for them or let me know?

File: docs/Documentation/Applications/vasp.md [hunk 4]
```diff
@@ -389,7 +383,7 @@ To run VASP, the following 4 input files are needed: POSCAR, POTCAR, INCAR, KPOI
     # If the multi-node calculations are breaking, replace the srun line with this line
     # I_MPI_OFI_PROVIDER=tcp mpirun -iface ens7 -np 60 vasp_std
     ```
-??? example "Vermilion: VASP 6 (Open MPI) on CPUs"
+??? example "Sample job script: Vermillion - VASP 6 (Open MPI)"
```
[the remark this answers is not in the document]
Comment (tayloraubry) [reply docs/Documentation/Applications/vasp.md:386]: I've created an issue to review all the vermillion modules and another to look into the number of tasks available.

File: docs/Documentation/Applications/vasp.md (hunk 1 above)
Comment (tayloraubry) [reply docs/Documentation/Applications/vasp.md:354]: I've created an issue to review all the vermillion modules.

File: docs/Documentation/Applications/vasp.md (hunk 2 above)
Comment (tayloraubry) [reply docs/Documentation/Applications/vasp.md:472]: I've created an issue to review all the vermillion modules.

**Comment by tayloraubry** [comment]
@mcfarljm Thanks for the review, I accepted all your suggestions and made several issues surrounding VASP on Vermillion scripts. 
@yandthj Let me know if I need to do anything else. (Not sure if I am supposed to close this)

**Review by mcfarljm — DISMISSED** [review]

**Review by yandthj — CHANGES_REQUESTED** [review]
Just noted a few spelling errors. 

I think it's an improvement to have removed the multiple links/explanations of the benchmark studies repo, but do you think there should be one link to it on this page? I'm not sure if that info is helpful to some users.

File: docs/Documentation/Applications/vasp.md [hunk 5]
```diff
@@ -1,117 +1,159 @@
...
 
-## Example Job Scripts
-### Eagle
+??? example "Build recommendations for VASP"
 
-??? example "Eagle: VASP 6 (Intel MPI) on CPUs"
+    We recommend building vasp with a full intel toolchain and launching with the cray-mpich-abi at runtime. Additionally, you should build on a compute node so that you have the same architecture as at runtime:
+
+    ```
+    salloc -N 1 -p standard -t <time> -A <account>
```
Comment (tayloraubry) [inline docs/Documentation/Applications/vasp.md:78]:
```suggestion
    salloc -N 1 -t <time> -A <account>
```

[the remark this answers is not in the document]
Comment (tayloraubry) [reply docs/Documentation/Applications/vasp.md]: Right, I think we used to need it on kestrel so sorry that was a carry over from friendly users phase. Fixed

**Review by yandthj — APPROVED** [review]

**Pull request merged**